src/main/java/racingcar/controller/Racing.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020 by 티거, 제이미
+ * All rights reserved.
+ *
+ * @author      티거, 제이미
+ * @version     1.0
+ * @date        12 Feb 2020
+ *
+ */
+
+package racingcar.controller;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import racingcar.domain.Car;
+import racingcar.domain.CarFactory;
+import racingcar.domain.RacingResult;
+import racingcar.domain.RacingResultFactory;
+import racingcar.util.Director;
+
+public class Racing {
+
+    private final static int RANDOM_NUMBER_RANGE = 10;
+
+    public static RacingResultFactory run(CarFactory carFactory, int tryCount) {
+        RacingResultFactory racingResultFactory = new RacingResultFactory();
+        for (int i = 0; i < tryCount; i++) {
+            playRacing(carFactory.getCars(), racingResultFactory);
+        }
+        return racingResultFactory;
+    }
+
+    private static void playRacing(List<Car> cars, RacingResultFactory racingResultFactory) {
+        LinkedHashMap<Car, Integer> racingResults = new LinkedHashMap<>();
```

Comment:
```suggestion
        Map<Car, Integer> racingResults = new LinkedHashMap<>();
```

위와 같이 수정한다면 어떠한 장점이 있을지 고민해보아요.
충분히 고민하셨다면 이펙티브 자바 `아이템 64. 객체는 인터페이스를 사용해 참조하라`를 참고해주세요!

Reply:
#### 생각한 장점
: 타입을 클래스가 아닌 인터페이스로 추상화한다면, 해당 인터페이스를 구현한 다른 클래스들도 선언한 변수에 값을 초기화(또는 대치)할 수 있습니다!

#### `아이템 64. 객체는 인터페이스를 사용해 참조하라`

- 인터페이스를 구현한 객체의 경우 인터페이스 타입으로 선언할 것

  - 단, 값 타입을 사용할 경우 클래스 타입을 사용

  - 단, 인터페이스에 없는 메서드를 사용해야 할 경우에는 클래스를 타입으로 선언할 것

- 인터페이스나 추상메서드를 타입으로 선언시

  - 참고 객체를 바꾸더라도(해당 타입을 구현 또는 상속한 클래스) 예외나 에러가 발생하지 않아 안전함

#### LinkedHashMap으로 선언했던 이유

 개념이 애매모호하게 머릿속에 들어있던 것 같습니다. LinkedHashMap의 경우엔 순서 보장을 해주는 Map이라서, Map으로 선언하게 되면 순서가 뒤엉킬거라고 착각을 한 나머지 LinkedHashMap으로 선언을 해버렸습니다.
 Map타입으로 LinkedHashMap 객체를 바라보는 것이기 떄문에 그런 문제는 발생하지 않는 것이네요 ㅠ_ㅠ 앞으로 더 열심히 공부하겠습니다!